uniffi_bindgen/src/bindings/python/gen_python.rs

```diff
@@ -141,9 +141,12 @@ mod filters {
             | Type::UInt64 => format!("int({})", nm), // TODO: check max/min value
             Type::Float32 | Type::Float64 => format!("float({})", nm),
             Type::Boolean => format!("bool({})", nm),
-            Type::String | Type::Object(_) | Type::Enum(_) | Type::Error(_) | Type::Record(_) => {
-                nm.to_string()
-            }
+            Type::String
+            | Type::Object(_)
+            | Type::Enum(_)
+            | Type::Error(_)
+            | Type::Record(_)
+            | Type::JSONObject => nm.to_string(),
```

Comment:
I'm mildly surprised that given the other comments above, JSONObject isn't kinda disguised more as an actual string. I'm sure a longer look will make it clearer though.